src/routes/solid-start/advanced/websocket.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: "Websocket Endpoint"
+---
+
+Websocket endpoint may be included by passing the ws handler file you specify in your start config. Note that this feature is experimental on the Nitro server and its config may change in future releases of Solid Start. Use it with caution.
```

Comment:
Please add a line break between the 2 sentences.

Also it's "SolidStart", not "Solid Start"

Could you add a link to the entry for the Nitro docs where it says the feature is experimental?